doc/faq.md

```diff
@@ -197,15 +197,26 @@ Cap'n Proto may be layered on top of an existing encrypted transport, such as TL
 
 ### How do I report security bugs?
 
-Please email [security@sandstorm.io](mailto:security@sandstorm.io).
+Please email [kenton@cloudflare.com](mailto:kenton@cloudflare.com).
 
 ## Sandstorm
 
 ### How does Cap'n Proto relate to Sandstorm.io?
 
-[Sandstorm.io](https://sandstorm.io) is an Open Source project and startup founded by Kenton, the author of Cap'n Proto. Cap'n Proto is owned and developed by Sandstorm the company and heavily used in Sandstorm the project.
+[Sandstorm.io](https://sandstorm.io) is an Open Source project and startup founded by Kenton, the author of Cap'n Proto. Cap'n Proto was developed by Sandstorm the company and heavily used in Sandstorm the project. Sandstorm ceased most operations in 2017 and formally dissolved as a company in 2022, but the open source project continues to be developed by the community.
 
 ### How does Sandstorm use Cap'n Proto?
 
 See [this Sandstorm blog post](https://blog.sandstorm.io/news/2014-12-15-capnproto-0.5.html).
 
+## Cloudflare
+
+### How does Cap'n Proto relate to Cloudflare?
+
+[Cloudflare Workers](https://workers.dev) is a next-generation cloud application platform. Kenton, the atuhor of Cap'n Proto, is the lead engineer on the Workers project. Workers heavily uses Cap'n Proto in its implementation, and the Cloudflare Workers team are now the primarily developers and maintainers of Cap'n Proto's primary C++ implementation.
```

Comment:
s/atuhor/author

Reply:
Fixed in 9a79822fd817af716f8c7a6fe753297d600b453f